angularjs-portal-home/src/main/webapp/partials/widgets/weather.html

```diff
@@ -1,5 +1,9 @@
 <div>
-    <loading-gif data-object='weatherData'></loading-gif>
+
+	<div id="loading">
```

Comment:
I suggest installing the eclipse java google styleguide

https://code.google.com/p/google-styleguide/source/browse/trunk/eclipse-java-google-style.xml